test/uk/gov/hmrc/helptosavefrontend/controllers/IvControllerSpec.scala

```diff
@@ -16,79 +16,56 @@
 
 package uk.gov.hmrc.helptosavefrontend.controllers
 
-import java.util.UUID
 import java.util.UUID.randomUUID
 
 import org.scalamock.scalatest.MockFactory
 import org.scalatest.prop.TableDrivenPropertyChecks._
 import org.scalatest.prop.Tables.Table
 import play.api.http.Status._
 import play.api.i18n.MessagesApi
-import play.api.mvc.Result
 import play.api.test.FakeRequest
-import uk.gov.hmrc.domain.Nino
+import uk.gov.hmrc.auth.core.AuthProvider.GovernmentGateway
+import uk.gov.hmrc.auth.core._
 import uk.gov.hmrc.helptosavefrontend.connectors.{IvConnector, SessionCacheConnector}
 import uk.gov.hmrc.helptosavefrontend.models.iv.{IvSuccessResponse, JourneyId}
-import uk.gov.hmrc.play.frontend.auth.connectors.AuthConnector
-import uk.gov.hmrc.play.frontend.auth.connectors.domain._
-import uk.gov.hmrc.play.frontend.auth.{AuthContext, AuthenticationProviderIds}
-import uk.gov.hmrc.play.http.{HeaderCarrier, SessionKeys}
+import uk.gov.hmrc.play.http.HeaderCarrier
 import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
-import uk.gov.hmrc.time.DateTimeUtils.now
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future}
 
 class IvControllerSpec extends UnitSpec with WithFakeApplication with MockFactory {
 
   val ivConnector: IvConnector = mock[IvConnector]
 
-  val jId = randomUUID().toString
+  private val jId = randomUUID().toString
```

Comment:
Is this `val` needed  - can you not just do `val journeyId = JounreyId(randomUUID().toString)`